lib/iris/_lazy_data.py

```diff
@@ -31,6 +33,19 @@
 import numpy.ma as ma
 
 
+def non_lazy(func):
```

Comment:
No such decorator exists, but depending on what you want to do, Array plugins might be useful:

http://dask.pydata.org/en/latest/array-creation.html#plugins

There is an example there for automatic computation.  Again, not exactly what I'm seeing here, but depending on the underlying problem you have this might help.

Reply:
Thanks for the suggestion @mrocklin! I wonder though if Array plugins aren't quite what we're looking for in this case.